vega_sim/scenario/common/agents.py

```diff
@@ -1581,22 +1582,30 @@ def initialise(
         self.vega.wait_for_total_catchup()
 
     def step(self, vega_state: VegaState):
+        market_info = self.vega.market_info(market_id=self.market_id)
+        market_in_auction = (
+            not market_info.trading_mode
+            == markets_protos.Market.TradingMode.TRADING_MODE_CONTINUOUS
+        )
```

Comment:
This should be contained within the vega_state object (though it needs updating to be clearer about what is inside)

Reply:
Could you offer the change here, I am not sure what you mean exactly.

Reply:
Added proposed change, but requires some other line tidying too

Reply:
Thx!